index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
+	</details>
 	</div>
 
 	<div class="req" id="loc_names_case_normalization">
 	<p class="advisement">Don't normalize the casing in names.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>Some names (such as 'McNamara') contain capital letters that are not the first letter; others (such as 'van der Waals') include words that are not capitalized. Forms should preserve the case the user enters and not coerce such names to always and only use capital letters at the start of each word.</p>
+
 	<div class="req" id="loc_names_spaces">
 	<p class="advisement">Allow the user to enter a name with spaces.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>Allows correct capture of a family name such as that of Gabriel García Márquez, or a given name such as José María Olazábal.</p>
+</section>
+
+
+
+
+
+
+<section id="loc_names_other">
+<h4>Other considerations</h4>
+
+
 	<div class="req" id="loc_names_same_names">
 	<p class="advisement">Don't assume that members of the same family will share the same family name.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>It would be wrong to assume that members of the same family share the same family name. There is a growing trend in the West for wives to keep their own name after marriage, but there are other cultures, such as China, where this is the normal approach. In some countries the wife may or may not take the husband's name.</p>
```

Comment:
Perhaps avoid considerations such as LGBT ones by genericizing "wife" and "husband", e.g.

> There is a growing trend in the West for individuals to keep their own name after marriage, but there are other cultures, such as China, where this is already the norm. In some countries a spouse may or may not choose to take the other's name (or they might do something else, such as hyphenating names)

Reply:
done